src/late_multicellular_stage/editor/MetaballBodyEditorComponent.cs

```diff
@@ -450,9 +497,16 @@ protected override MetaballLayout<MulticellularMetaball> CreateLayout()
         return new MetaballLayout<MulticellularMetaball>(OnMetaballAdded, OnMetaballRemoved);
     }
 
-    protected override IMetaballDisplayer<MulticellularMetaball> CreateMetaballDisplayer()
+    protected override IMetaballDisplayer<MulticellularMetaball> CreateVisualMetaballDisplayer()
     {
-        var displayer = metaballDisplayerScene.Instantiate<MulticellularMetaballDisplayer>();
+        var displayer = visualMetaballDisplayerScene.Instantiate<MulticellularConvolutionDispayer>();
```

Comment:
Ah, this seems to be where I realize that visual metaball displayer is the convolution displayer.
I guess the variable naming is fine but I'll still ask for those extra comments to explain things to anyone new stumbling upon the variables.

Reply:
I added the comments (specifically clarified that the visual displayer displays convolution surfaces)